Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -32,6 +32,16 @@
 var environmentMapURL = '../../SampleData/EnvironmentMap/kiara_6_afternoon_2k_ibl.ktx';
 var modelURL = '../../SampleData/models/Spheres/MetalRoughSpheres.glb';
 
+// CesiumJS supports image-based lighting through the glTF extension `EXT_lights_image_based`.
+// Until more tools can support this extension, the environment map can be set by manually
```

Comment:
OK, soap-box time.  PBR, and in particular PBR metallic materials, don't work well with only analytic or punctual lights.  IBL is required for them to look OK.  Cesium has a set of procedural gradients representing reflected light from the Earth that is used as a (poor?) substitute for IBL, when captured IBL environments aren't available, which is how the master branch is able to show glTF 2.0 without IBL image loading.

I fully expect all glTF viewers to have the option to "manually" load IBL environment maps.  This isn't some stop-gap measure while the details of an IBL extension are hammered out.  This isn't a case where soon we expect all normal glTFs to naturally carry around the baggage of full IBL maps on their insides.  We do not, and should not.  The default, normal behavior, is that the user should select a glTF, and separately select an IBL to go with the situation they've placed their glTF inside of.  The two don't come as a pair.

Apologies if my wording is too strong here.  I've talked to people about this, and written about it, in several different places ever since the concept of an IBL extension to glTF was first floated a year ago.  My worry is that developers are going to rely on this extension as the primary means to deliver IBL with glTF, when in my mind it's vital to offer that separation to end users, to make the user choose IBL separately from choosing a glTF.  Additionally I worry that documentation and samples are going to be written such that they steer new users towards this concept that the IBL must be embedded and deeply paired with the glTF.

Anyway, I guess this is too long a review comment for a simple wording change to a code sample comment.  But it would be great to help educate users that they are encouraged to load their own IBLs, and try out their models in a number of different IBLs to see what looks best in a given location in Cesium's scene.

Reply:
> This isn't a case where soon we expect all normal glTFs to naturally carry around the baggage of full IBL maps on their insides.

I was expecting that, but keeping these options around to change the IBL sounds good to me. I was thinking we should have an option on the scene to override the default procedural IBL as well.